### DISCUSSION TITLE
[Script request] LXC-iptag

### DISCUSSION THREAD

**Discussion by lasharor** [community-scripts/ProxmoxVE#395] [Request script]
### Application Name

LXC-IPtag

### Website

https://github.com/gitsang/lxc-iptag

### Description

lxc-iptag is a simple script to add ip tags to LXC containers.

I think this should be part of the proxmox VE Tools section. 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
@gitsang
I'm not sure if you're reading this.
Do you have any concerns about integrating your script into the Proxmox community project?

**Reply by gitsang** [reply]
Thank you for asking.

I have no concerns and glad that the community is interested in integrating this script.

**Reply by lasharor** [reply]
Sorry, I didn't know I could tag you in this request. @gitsang

**Reply by gitsang** [reply]
No worries. 😄 @lasharor
